packages/astro/test/astro-class-list.nodetest.js

```diff
@@ -0,0 +1,63 @@
+import assert from 'node:assert/strict';
+import { describe, before, it } from 'node:test';
+import * as cheerio from 'cheerio';
+import { loadFixture } from './test-utils.js';
+
+describe('Class List', async () => {
+	let fixture;
+
+	before(async () => {
+		fixture = await loadFixture({ root: './fixtures/astro-class-list/' });
+		await fixture.build();
+	});
+
+	it('Passes class:list attributes as expected to elements', async () => {
+		const html = await fixture.readFile('/index.html');
+		const $ = cheerio.load(html);
+
+		assert.strictEqual($('[class="test control"]').length, 1, '[class="test control"]');
```

Comment:
`assert.equal` is an alias of `assert.strictEqual`, because we are importing `node:assert/strict`. So we can use `assert.equal`, like we are doing in all tests. Would you mind changing it?